account_move_line_sequence/views/account_move_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_move_form__ext" model="ir.ui.view">
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='line_ids']/tree//field[1]"
+                position="before"
+            >
+                <field name="sequence_journal_item" widget="handle" />
```

Comment:
I'm really not getting why this widget=handle does not trigger. Maybe it's a quirk of the migration instance, anyway, the dragging works so I don't want to spend a lot of time on it.

Reply:
It doesn´t? It didn't work before because I erroneously overwrote the sequence field. Now that sequence_journal_item field is there, the widget also works